packages/contracts-bedrock/contracts/deployment/SystemDictator.sol

```diff
@@ -307,7 +316,7 @@ contract SystemDictator is OwnableUpgradeable {
         config.globalConfig.proxyAdmin.upgradeAndCall(
             payable(config.proxyAddressConfig.optimismPortalProxy),
             address(config.implementationAddressConfig.optimismPortalImpl),
-            abi.encodeCall(OptimismPortal.initialize, ())
+            abi.encodeCall(OptimismPortal.initialize, (optimismPortalDynamicConfig))
```

Comment:
Small nit- Generally like to avoid `abi.encodeCall` due to the extra type checking added at runtime. We know that `optimismPortalDynamicConfig` is a bool, so there's no real need.

Reply:
Oh interesting, I thought that the type checking happened at compile time for `encodeCall`

Reply:
Ah actually you're right about this- mb